spec/routing/v1/doorkeeper/authorize_routing_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+RSpec.describe 'routing', type: :routing do
+  it 'routes to #new' do
+    expect(get: '/v1/oauth/authorize').to route_to('v1/doorkeeper/authorizations#new')
+  end
+
+  it 'routes to #show' do
+    expect(get: '/v1/oauth/authorize/native').to route_to('v1/doorkeeper/authorizations#show')
+  end
+
+  it 'routes to #destroy' do
+    expect(delete: '/v1/oauth/authorize/').to route_to('v1/doorkeeper/authorizations#destroy')
+  end
+
+  it 'routes to #create' do
+    expect(post: '/v1/oauth/authorize').to route_to('v1/doorkeeper/authorizations#create')
+  end
+
+  context 'when no API version is specified' do
+    it 'routes to #create' do
+      expect(get: '/oauth/authorize').to route_to('v1/doorkeeper/authorizations#new')
+    end
```

Comment:
I think this is a duplicate of the test on lines 29-31. You can remove one of them. 

If you keep this one then the description should to be changed to `it 'routes to #new' do`, as you already have a test for `#create` on line 37.